### DISCUSSION TITLE
Nodes emit iBeacon?

### DISCUSSION THREAD

**Discussion by manunited10** [mKeRix/room-assistant#1367] [Q&A] [answered]
Hi. I have two room-assistant nodes on raspberry pi zero w.
A while ago I enabled `bluetoothLowEnergy` integration for some tests and ended up leaving the integration there and not having any ble devices. 
Today I noticed that my nodes `emit` `iBeacon` and my home assistant iBeacon integration can pick them. Wondering if that's an expected behaviour (to be a transmitter)?
Thanks.

**Comment by lorenzothurman** [comment, marked as answer]
That might have to do with having instanceBeaconEnabled set to true. Check the docs on it.
